integration-tests/bats/sql-local-remote.bats

```diff
@@ -431,6 +431,62 @@ EOF
   stop_sql_server 1
 }
 
+@test "sql-local-remote: verify dolt conflicts cat behavior" {
+  cd defaultDB
+
+  export DOLT_CLI_PASSWORD=""
```

Comment:
Drop this in this file, and remove all the `--user dolt` options. This can be a fast follow change

Reply:
Good catch, updating.

Reply:
Removed.